pkg/controllers/dynakube/phase.go

```diff
@@ -187,9 +187,9 @@ func (controller *Controller) numberOfMissingActiveGatePods(dk *dynakube.DynaKub
 	sum := int32(0)
 	capabilityFound := false
 
-	for _, activeGateCapability := range capabilities {
+	for range capabilities {
```

Comment:
the for loop is not needed here, `capabilities` is always a slice containing 1 item.
It can be a next step to refactor `GenerateActiveGateCapabilities` but why to iterate over a slice and ignore items?

Reply:
I had to stop at some point with my refactor 😅

Reply:
thx :smile: